service-print/src/main/java/org/oskari/print/PDF.java

```diff
@@ -137,6 +147,9 @@ public class PDF {
     private static final String MARKER_COORD_SEPARATOR= "_";
 
     private static final FilterFactory2 ff = CommonFactoryFinder.getFilterFactory2(null);
+    
+    private static final ResourceBundle rb = ResourceBundle.getBundle("messages");
```

Comment:
We should rename "messages" to something else to avoid confusion when theres another localization file set called messages. Maybe just rename to "print-messages"?